script/preview.js

```diff
@@ -97,6 +97,7 @@ app.get('/preview', async (req, res, next) => {
   try {
     const smith = await createPipieline({
       ...options,
+      isPreviewServer: true,
```

Comment:
We need a flag to send to `process-entry-names` to indicate that we should not try to do that whole TeamSite step. This was how the old plugin `addAssetHashes` worked 🤷‍♂